### PR TITLE
support phpMyAdmin configuration storage settings

### PR DESCRIPTION
**Pull request by LeonPoon**
This is 2nd attempt. (First one is #34, closed because I didn't have time to respond to the latest suggestions...)

Solves #27.

Am not wanting to use zeroconf, so I need the PMA storage settings.

A note about `create_tables.sql`, I would make a choice to not handle creating the tables here. Typically database side of things are set up outside of phpmyadmin setup. Other than this, I did:

* rename the env vars to start with `CONFIGURATION_STORAGE_` and gave some sane default values, and
* remove the need to code in a list of table names.

### CODE REVIEW

File: 5/debian-10/rootfs/opt/bitnami/scripts/libphpmyadmin.sh [hunk 1]
```diff
@@ -91,6 +91,19 @@ phpmyadmin_initialize() {
         ! is_empty_value "$DATABASE_SSL_VERIFY" && phpmyadmin_conf_set "\$cfg['Servers'][\$i]['ssl_verify']" "$(php_convert_to_boolean "$DATABASE_SSL_VERIFY")" yes
     fi
 
+    if ! is_empty_value "$CONFIGURATION_STORAGE_DATABASE_CONTROLUSER" &&
+       ! is_empty_value "$CONFIGURATION_STORAGE_DATABASE_CONTROLPASS" &&
+       ! is_empty_value "$CONFIGURATION_STORAGE_DATABASE_CONTROLHOST" &&
+       ! is_empty_value "$CONFIGURATION_STORAGE_DATABASE_CONTROLPORT" &&
+       ! is_empty_value "$CONFIGURATION_STORAGE_DATABASE_PMADB"; then
```
Comment (migruiz4) [inline 5/debian-10/rootfs/opt/bitnami/scripts/libphpmyadmin.sh:98]: I think that ignoring everything if a single variable is missing may lead to confusion.

I would suggest adding a variable, e.g `CONFIGURATION_STORAGE_ENABLE` that made it easier to control:

```suggestion
    if is_boolean_yes "$CONFIGURATION_STORAGE_ENABLE"; then
```

Then, in the `phpmyadmin_validate`, add a section like this:
```bash
    if is_boolean_yes "$CONFIGURATION_STORAGE_ENABLE"; then
        for empty_env_var in "CONFIGURATION_STORAGE_DATABASE_CONTROLHOST" "CONFIGURATION_STORAGE_DATABASE_CONTROLPORT" "CONFIGURATION_STORAGE_DATABASE_CONTROLUSER" "CONFIGURATION_STORAGE_DATABASE_CONTROLPASS"
"CONFIGURATION_STORAGE_DATABASE_PMADB"; do
            is_empty_value "${!empty_env_var}" && print_validation_error "The ${empty_env_var} environment variable is empty or not set."
        done
  fi
```

**Review by migruiz4 — CHANGES_REQUESTED** [review]
Hi @LeonPoon,

Thank you very much for your contribution! Please check my comment below.

File: 5/debian-10/rootfs/opt/bitnami/scripts/libphpmyadmin.sh (hunk 1 above)
Comment (LeonPoon) [reply 5/debian-10/rootfs/opt/bitnami/scripts/libphpmyadmin.sh]: Done that. Please review.

**Review by javsalgar — APPROVED** [review]
LGTM

**Comment by juan131** [comment]
Hi @LeonPoon 

A new container was released including your changes (version `5.1.1-debian-10-r166`). Please note that some env. variables were renamed to follow the naming standards we usually follow, find more info in the commit below:

- https://github.com/bitnami/bitnami-docker-phpmyadmin/commit/8ba2a4ef723e5b280afcb30ab552ba6812ced01e

**Pull request merged**